src/GitHub.App/Infrastructure/LogManager.cs

```diff
@@ -28,7 +28,10 @@ static Logger CreateLogger()
 
         static Lazy<Logger> Logger { get; } = new Lazy<Logger>(CreateLogger);
 
+        //Violates CA1004 - Generic methods should provide type parameter
+        #pragma warning disable CA1004
         public static ILogger ForContext<T>() => Logger.Value.ForContext<T>();
```

Comment:
So this threw me for a loop originally, but CA1004 is an error by default and we can only pragma disable warnings.